backends-common/elasticsearch/src/main/java/org/apache/james/backends/es/ElasticSearchConfiguration.java

```diff
@@ -128,6 +129,21 @@ static SSLValidationStrategy from(String rawValue) {
             }
         }
 
+        public enum HostNameVerifier {
+            DEFAULT,
+            NOOP;
```

Comment:
I guess I prefer ACCEPT_ANY_HOST name as an option, it makes the intent clearer.

Reply:
> If you can remove the last commit by configuring correctly hostnames in certificates

I don't know much about the SSL things, so I didn't think about that solution. Even when it's a good suggestion, there are two concerns for me, using default hostname verifier everytime will break the test of ignoring SSL check, second, I'm not sure the docker CI behave in the same way as local. It doesn't return hostname as localhost but an IP address. That's what I observed several days ago.

Reply:
> > If you can remove the last commit by configuring correctly hostnames in certificates
> 
> I don't know much about the SSL things, so I didn't think about that solution. Even when it's a good suggestion, there are two concerns for me, using default hostname verifier everytime will break the test of ignoring SSL check

Yes, but it's a test issue, but I understand this point and that's why I accept this solution.

> , second, I'm not sure the docker CI behave in the same way as local. It doesn't return hostname as localhost but an IP address. That's what I observed several days ago.

An hostname verifier works also with IP when they are in the SubjectAltName field of the certificate.